components/suggest/src/suggest.udl

```diff
@@ -40,6 +40,7 @@ interface Suggestion {
         string url,
         string raw_url,
         sequence<u8>? icon,
```

Comment:
It might be time to add a struct for the icon, like:

```webidl
dictionary SuggestionIcon {
    sequence<u8> icon;
    string mime_type;
}
```

And then in Rust:

```rs
struct SuggestionIcon {
    icon: Vec<u8>,
    mime_type: String,
}
````

...But only if you want to; the way you have it also looks good to me! 😊

Reply:
Yeah, I also thought about it. Please let me do it in another PR!